frontend/src/scenes/trends/ActionFilter/ActionFilter.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { entityFilterLogic, EntityTypes } from './actionFilterLogic'
+import { Link } from 'react-router-dom'
+import { ActionFilterRow } from './ActionFilterRow'
+import { capitalizeFirstLetter } from '~/lib/utils'
+
+export function ActionFilter(props) {
```

Comment:
In terms of UX, I think it would be better if everything was just in one list, rather than having the three bolded bits of text (actions/events/unspecified). People will be able to tell by clicking into it which one it is, and we could always add it a bit more subtly on the right as we've got plenty of space. I think the way it currently works lots of things jump around which doesn't make for an A+ experience. Not far off though!

Reply:
fixed!